src/main/java/com/elixr/poc/constant/ErrorConstants.java

```diff
@@ -0,0 +1,13 @@
+package com.elixr.poc.constant;
+
+/* this constant file holds all the error messages for all the end points */
+
+public class ErrorConstants {
+    private ErrorConstants() {}
+
+    public static final String FIRST_NAME_MISSING_IN_USER = "First name is mandatory";
+    public static final String USER_NAME_MISSING_IN_USER = "User name is mandatory";
+    public static final String LAST_NAME_MISSING_IN_USER = "Last name is mandatory";
+
+
```

Comment:
Remove this empty line

Reply:
Removed the empty lines and formatted propperly